gremlin-go/driver/client_test.go

```diff
@@ -40,13 +41,13 @@ func TestClient(t *testing.T) {
 				settings.AuthInfo = testNoAuthAuthInfo
 			})
 		defer client.Close()
-		assert.Nil(t, err)
+		assert.NoError(t, err)
```

Comment:
Errcheck assertion is traditionally done with `assert.NoError`. There are couple more files with this slight inconvenience which I chose not to touch for now.